backend/dal/Configuration/AccessRequestConfiguration.cs

```diff
@@ -35,8 +34,8 @@ public override void Configure(EntityTypeBuilder<AccessRequest> builder)
             builder.HasOne(m => m.Status).WithMany(u => u.AccessRequests).HasForeignKey(m => m.StatusId).OnDelete(DeleteBehavior.Cascade).HasConstraintName("PIM_ARQSTT_PIM_ACRQST_FK");
 
             builder.HasMany(m => m.Organizations).WithMany(m => m.AccessRequests).UsingEntity<AccessRequestOrganization>(
-                m => m.HasOne(m => m.Organization).WithMany(m => m.AccessRequestsManyToMany).HasForeignKey(m => m.OrganizationId),
-                m => m.HasOne(m => m.AccessRequest).WithMany(m => m.OrganizationsManyToMany).HasForeignKey(m => m.AccessRequestId)
+                m => m.HasOne(m => m.Organization).WithMany(m => m.AccessRequestsManyToMany).HasForeignKey(m => m.OrganizationId).OnDelete(DeleteBehavior.ClientCascade),
```

Comment:
Are these foreign keys named correctly?

Reply:
PIM_ACRQST_PIM_ACRQOR_FK and PIM_ORG_PIM_ACRQOR_FK